packages/nextjs-kit/src/components/previewRibbon.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+// import Image from 'next/image';
+
+interface PreviewRibbonProps {
+	clearPreviewRoute?: string;
+}
+
+const PreviewRibbon: React.FC<PreviewRibbonProps> = ({
+	clearPreviewRoute = '/api/clear-preview',
+}) => {
+	const [show, setShow] = useState(true);
+
+	return (
+		<div>
+			<div
+				className={`ps-flex ps-justify-between ps-items-center ps-min-h-[3.5rem] ps-h-14 ps-px-8 ps-py-2 ps-bg-[#3017A1] ps-text-white ps-transition-all ps-duration-300 ps-w-full ${
+					!show
+						? 'ps-justify-start ps-translate-x-[calc(100%-5rem)] ps-pl-4'
+						: ''
+				}`}
+			>
```

Comment:
I'm not sure exactly what changed, but when the preview banner is closed, I get the left-right scroll bar and if I scroll I see the part of the banner that is supposed to be hidden.

![image](https://user-images.githubusercontent.com/63015754/204378623-a2e26127-0233-4123-acd4-ce5ed8f17bff.png)

Reply:
Was because the Layout component in drupal has overflow-x-hidden, so I'll add it to the wordpress one